crates/wash-lib/src/config.rs

```diff
@@ -43,6 +44,10 @@ pub fn downloads_dir() -> Result<PathBuf> {
     Ok(cfg_dir()?.join(DOWNLOADS_DIR))
 }
 
+pub fn default_registry_credentials_file() -> Result<PathBuf> {
+    Ok(cfg_dir()?.join(DEFAULT_REGISTRY_CREDS_FILE))
```

Comment:
Hmm, I think we might want to associate the creds file with a context, instead of having a global one. What do you think?